### PR TITLE
Fix jsp_defaultHTML test to also check for 302

### LINKED ISSUE #28629: Fix  `jsp_defaultHTML` test
RTC 300373

```jsp_defaultHTML:junit.framework.AssertionFailedError: 2024-06-01-11:15:13:357 null
    at io.openliberty.http.monitor.fat.JSPApplicationTest.jsp_defaultHTML(JSPApplicationTest.java:117)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
    at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:203)
    at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:363)
    at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:177)
```

Sometimes 200, sometimes 302 when loading the default page configured in web.xml.

If a 200 isn't present, check for a 302.

### PR DESCRIPTION
**Pull request by Channyboy**
#build
fixes #28629

### CODE REVIEW

File: dev/io.openliberty.http.monitor_fat/fat/src/io/openliberty/http/monitor/fat/JSPApplicationTest.java [hunk 1]
```diff
@@ -123,7 +123,14 @@ public void jsp_defaultHTML() throws Exception {
...
         res = requestHttpServlet(route, server, requestMethod);
 
-        assertTrue(validatePrometheusHTTPMetric(getVendorMetrics(server), expectedRoute, responseStatus, requestMethod));
+        /*
+         * If 200 isn't present. The server may have redirected the request to the default page
+         * and issued a 302.
+         */
+        if (!validatePrometheusHTTPMetric(getVendorMetrics(server), expectedRoute, responseStatus, requestMethod)) {
+            responseStatus = "302";
+            validatePrometheusHTTPMetric(getVendorMetrics(server), expectedRoute, responseStatus, requestMethod);
```
Comment (fmhwong) [inline dev/io.openliberty.http.monitor_fat/fat/src/io/openliberty/http/monitor/fat/JSPApplicationTest.java]: Is `assertTrue` missing here?

**Review by fmhwong — CHANGES_REQUESTED** [review]
Need a check for success or fail for this test case.

File: dev/io.openliberty.http.monitor_fat/fat/src/io/openliberty/http/monitor/fat/JSPApplicationTest.java (hunk 1 above)
Comment (Channyboy) [reply dev/io.openliberty.http.monitor_fat/fat/src/io/openliberty/http/monitor/fat/JSPApplicationTest.java]: That would be important.

**Comment by Channyboy** [comment]
#build #libby

**Review by fmhwong — COMMENTED** [review]
LGTM

**Review by fmhwong — APPROVED** [review]
LGTM

**Pull request merged**